web/src/app/loading/components/LoadingButton.tsx

```diff
@@ -1,9 +1,29 @@
 import React from 'react'
-import p from 'prop-types'
-import Button from '@mui/material/Button'
+import Button, { ButtonPropsColorOverrides } from '@mui/material/Button'
 import CircularProgress from '@mui/material/CircularProgress'
+import { OverridableStringUnion } from '@mui/types'
 
-const LoadingButton = (props) => {
+interface LoadingButtonProps {
```

Comment:
it might be easier to extend Button props so we don't have to define things like `color` or `onClick` by hand

`interface LoadingButtonProps extends ButtonProps {`